src/cmd-fetch

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Comment:
Testing this out now, but one thing I can definitely say is that this file is missing execute :zap:  bits.